react/MuiCozyTheme/ExpansionPanelSummary/theme.js

```diff
@@ -0,0 +1,36 @@
+const MuiExpansionPanelSummary = {
+  expanded: {},
+  root: {
+    backgroundColor: 'var(--paleGrey)',
+    textTransform: 'uppercase',
```

Comment:
I'm not sure this is a default value.

Reply:
You may be right, but since we don't have another example with another design for now, I think we can keep it.